gradle/libs.versions.toml

```diff
@@ -10,7 +10,7 @@ gatling = "3.8.4"
 googleJavaFormat = "1.15.0"
 guava = "31.1-jre"
 hadoop = "3.3.4"
-iceberg = "1.0.0"
+iceberg = "1.1.0"
```

Comment:
Do we need to update `nessieClientVersion` from `0.30.0` to `0.44.0` as part of this?